sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Program.cs

```diff
@@ -42,13 +42,15 @@ public static async Task Main(string[] args)
 
             // Instantiate the client
 
+            // These environment variables are necessary for application Id to login using a client secret.
+            Environment.SetEnvironmentVariable("AZURE_CLIENT_SECRET", options.ClientSecret);
+            Environment.SetEnvironmentVariable("AZURE_CLIENT_ID", options.ClientId);
+            Environment.SetEnvironmentVariable("AZURE_TENANT_ID", options.TenantId);
+
             var httpClient = new HttpClient();
             DigitalTwinsClient dtClient = (options.GetLoginMethod()) switch
             {
-                LoginMethod.AppId => GetDigitalTwinsClient(
-                    options.TenantId,
-                    options.ClientId,
-                    options.ClientSecret,
+                LoginMethod.AppId => GetDigitalTwinsClient(                   
                     options.AdtEndpoint),
```

Comment:
So we still set this as arguments to sample? But others as env vars?

Reply:
Would you not need to remove these options from Options class?

Reply:
We are getting the values from options class and setting to env variables so we don't need to check in any secret values.